src/sdk/bucketspace_fs.js

```diff
@@ -648,7 +649,7 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
     // TODO: move the following 3 functions - has_bucket_action_permission(), validate_fs_bucket_access(), _has_access_to_nsfs_dir()
     // so they can be re-used
     async has_bucket_action_permission(bucket, account, action, bucket_path = "") {
-        const account_identifier = account.name.unwrap();
+        const account_identifier = account.name?.unwrap();
```

Comment:
I think now this is not strictly needed right? because the anonymous user will have an identifier.